python/ray/dag/tests/experimental/test_torch_tensor_dag.py

```diff
@@ -415,128 +415,128 @@ def destroy(self) -> None:
-            return self._world_size
-
-        def get_self_rank(self) -> Optional[int]:
-            return self._rank
-
-        def get_actor_handles(self) -> List["ray.actor.ActorHandle"]:
-            return self._actor_handles
-
-        def send(self, value: "torch.Tensor", peer_rank: int) -> None:
-            return None
-
-        def recv(
-            self,
-            shape: Tuple[int],
-            dtype: "torch.dtype",
-            peer_rank: int,
-            allocator: Optional[TorchTensorAllocator] = None,
-        ) -> "torch.Tensor":
-            return None
-
-        def allreduce(
-            self,
-            send_buf: "torch.Tensor",
-            recv_buf: "torch.Tensor",
-            op: ReduceOp,
-        ) -> None:
-            raise NotImplementedError
-
-        def destroy(self) -> None:
-            pass
-
-    nccl_group = MockNcclGroup(2, [actor1, actor2])
-
-    # Mixed usage of NCCL groups should throw an error
-    # Case 1: custom NCCL group first, then default NCCL group
-    with InputNode() as inp:
-        dag = actor1.send_with_tuple_args.bind(inp)
-        dag = dag.with_type_hint(TorchTensorType(transport=nccl_group))
-        dag = actor2.recv.bind(dag)
-        dag = actor2.send_with_tuple_args.bind(dag)
-        dag = dag.with_type_hint(TorchTensorType(transport="nccl"))
-        dag = actor1.recv.bind(dag)
-    with pytest.raises(
-        ValueError,
-        match=r"Accelerated DAGs do not support mixed usage of type hints.*",
-    ):
-        dag.experimental_compile()
-
-    # Case 2: default NCCL group first, then custom NCCL group
-    with InputNode() as inp:
-        dag = actor1.send_with_tuple_args.bind(inp)
-        dag = dag.with_type_hint(TorchTensorType(transport="nccl"))
-        dag = actor2.recv.bind(dag)
-        dag = actor2.send_with_tuple_args.bind(dag)
-        dag = dag.with_type_hint(TorchTensorType(transport=nccl_group))
-        dag = actor1.recv.bind(dag)
-    with pytest.raises(
-        ValueError,
-        match=r"Accelerated DAGs do not support mixed usage of type hints.*",
-    ):
-        dag.experimental_compile()
-
-    nccl_group2 = MockNcclGroup(2, [actor1, actor2])
-
-    # Using two different custom NCCL groups are currently not supported
-    with InputNode() as inp:
-        dag = actor1.send_with_tuple_args.bind(inp)
-        dag = dag.with_type_hint(TorchTensorType(transport=nccl_group))
-        dag = actor2.recv.bind(dag)
-        dag = actor2.send_with_tuple_args.bind(dag)
-        dag = dag.with_type_hint(TorchTensorType(transport=nccl_group2))
-        dag = actor1.recv.bind(dag)
-    with pytest.raises(
-        ValueError,
-        match=(
-            "Accelerated DAGs currently only support "
-            "a single custom NCCL group, but multiple "
-            "have been specified."
-        ),
-    ):
-        dag.experimental_compile()
+# @pytest.mark.parametrize("ray_start_regular", [{"num_cpus": 4}], indirect=True)
```

Comment:
Leave a comment on why this is commented out.

Reply:
Done.